src/test/java/outcomes/process_executor/go/no_find_executable_file_by_directory/NoFindExecutableFileByDirectory.java

```diff
@@ -0,0 +1,27 @@
+package outcomes.process_executor.go.no_find_executable_file_by_directory;
+
+import org.hyperskill.hstest.common.FileUtils;
+import org.hyperskill.hstest.dynamic.input.DynamicTestingMethod;
+import org.hyperskill.hstest.testcase.CheckResult;
+import org.hyperskill.hstest.testing.TestedProgram;
+import outcomes.base.ContainsMessage;
+import outcomes.base.UserErrorTest;
+
+
+public class NoFindExecutableFileByDirectory extends UserErrorTest {
+
+    @ContainsMessage
+    String m =
+            "Error in test #1\n" +
+                    "\n" +
+                    "Cannot find a file to execute your code in directory \"" + FileUtils.cwd() +
```

Comment:
I prefer not to use `cwd` in checking feedback because they will work only in Windows or only in Linux but not in both. It's because of different slashes used in filesystem - `/` in Linux and `\` in Windows. 

I recommend doing two checks separately:
```java
    @ContainsMessage
    String m1 =
            "Error in test #1\n" +
            "\n" +
            "Cannot find a file to execute your code in directory \"";

    @ContainsMessage
    String m2 = "no_find_executable_file_by_directory";
```